client/app/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { CtaTypes } from "~/enums/CtaTypes";
+
+interface ButtonProps { 
+    text: string;
+    type: CtaTypes;
+    isSubmit?: boolean;
+}
```

Comment:
ar trebui sa folosim type in loc de interface. Nu mai e considerat good practice sa folosim oop concepts 

```suggestion
type ButtonProps { 
    text: string;
    type: CtaTypes;
    isSubmit?: boolean;
}
```